src/UglyToad.PdfPig/Content/BasePageFactory.cs

```diff
@@ -118,7 +118,7 @@ public abstract class BasePageFactory<TPage> : IPageFactory<TPage>
             }
             else if (DirectObjectFinder.TryGet<ArrayToken>(contents, PdfScanner, out var array))
             {
-                using var bytes = new ArrayPoolBufferWriter<byte>();
+                using var bytes = new ArrayPoolBufferWriter<byte>(1024 * 64);
```

Comment:
Update - the increase memory usage seems to not come from here, but my question stands. Thanks!

Reply:
I had set the initial buffer size in this case to cover the observed object sizes in the test suite. Most are in the 4KB range, but I saw some object sizes around 40KB. 

Otherwise, a 50KB segment has 8 additional copy operations, and array pool rentals, returns. 

e.g. 256 -> 512 > 1024 > 2048 -> 4096 -> 8192 -> 16384 -> 32768 -> 65536

We're likely to see a small regression on CPU time utilizing the pool, but much better steady state performance as we reduce the GC pressure.